classes/pdf/HTMLTemplateInvoice.php

```diff
@@ -80,7 +88,7 @@ public function getHeader()
     /**
      * Compute layout elements size.
      *
-     * @param $params Array Layout elements
+     * @param array $params Layout elements
```

Comment:
👍

Reply:
It can be changed, the signature is `mixed` not array :thinking:

Reply:
Can or not ?

Reply:
Can't sorry forget a part :D